docs/api/sampler.md

```diff
@@ -117,6 +118,7 @@ Sampler. Rules with `Numpy` in their name can only be used with
   rules.LocalRule
   rules.CustomRuleNumpy
   rules.ExchangeRule
+  rules.ParticleExchangeRule
```

Comment:
You could also add below a block like the following, which duplicates a bit the docs but maybe would help searchability

```
## Experimental models

The following samplers are experimental, meaning that we could change them at some point, and we actively seeking for feedback and opinions on their usage and APIs.


### Particle-specific samplers

The following models are for 2nd-quantisation fermionic hilbert spaces ({class}`netket.experimental.hilbert.SpinOrbitalFermions`).

```{eval-rst}
.. currentmodule:: netket.experimental.sampler

```

```{eval-rst}
.. autosummary::
   :toctree: _generated/samplers
   :template: flax_module_or_default
   :nosignatures:


   MetropolisParticleExchange
```

And the corresponding rules
```{eval-rst}
.. autosummary::
   :toctree: _generated/samplers
   :template: flax_module_or_default
   :nosignatures:


   rules.ParticleExchangeRule
```